### PR TITLE
New command + new MultipleAchievement

### PR DESCRIPTION
**Pull request by Phoetrix**
Hi! :)

I would like to add a category "Customs Achievements" to the plugin. This category can contain multiple custom categories and there is nothing listener attached to this new category.
The only way to increase the progression of customs achievements is by using my new command.

Why I do that? Because a lot of plugins allow server-owner to set up a list of custom commands that would be executed by the plugin when an event is called.
So we can set up my new command on plugins that allow the use of custom commands to increase the progression of a custom achievement.

Example: When a player vote for a server (on a Minecraft vote website), the server execute often a command to send a reward to a player. We can easily add my new command to increase by 1 the value of the Custom Achievement "Votes".

My new command is /aach add <amount> <category> <player>
This command increase the progression of an achievement with the value in "amount" argument. It work with all categories (except Connections?) but I write it principally for Customs Achievements.
For multiple achievement, you must to write the category like that: category.subcategory
Example: /aach add 1 places.dirt phoetrix

I am sorry in advance if my class "AddCommand.java" is not good. I finded one only way to increase the value of achievements in a non-listener class. I added a class in the class. :/

### CODE REVIEW

File: src/main/java/com/hm/achievement/command/HelpCommand.java [hunk 1]
```diff
@@ -101,6 +103,10 @@ public void extractConfigurationParameters() {
 								new String[] { "ACH", "NAME" }, new String[] { "&oach&7", "&oplayer&7" }));
 		langCommandGiveHover = plugin.getPluginLang().getString("aach-command-give-hover",
 				"Player must be online; only Commands achievements can be used.");
+		langCommandAdd = plugin.getChatHeader() + configColor + "/aach add &oamount cat player" + ChatColor.GRAY + " > "
+				+ ChatColor.translateAlternateColorCodes('&', "Increase the progression of an achievement.");
+		langCommandAddHover = plugin.getPluginLang().getString("aach-command-add-hover",
+				"Player must be online; All achievements can be used except Commands achievements; Write a subcategory like this: category.subcategory");
```
Comment (PyvesB) [inline src/main/java/com/hm/achievement/command/HelpCommand.java]: This is too long as well. Recent Minecraft versions handle multi-line hovers, but this isn't the case on older ones. I just tried launching version 1.8.4, with my screen resolution I can only see `Write a subcategory like this: category.subcategory`, the rest is hidden! 😄 

A simple `Player must be online; mainly used for Customs achievements.` may do the trick. We can add more details to the Wiki.

File: src/main/resources/config.yml [hunk 2]
```diff
@@ -623,6 +623,16 @@ PlayerCommands:
         Commands:
           Execute: "say §cALERT: §7PLAYER is now a statistician"
           Display: Broadcast
+          
+# When the command /aach add is called for this category.
+Customs:
+  votes:
```
Comment (PyvesB) [inline src/main/resources/config.yml]: `votes` is definitely a good example, several users have requested voting support in the past! 👍

File: src/main/resources/gui.yml [hunk 3]
```diff
@@ -170,9 +170,12 @@ PlayerCommands:
 Commands:
   Item: bookshelf
   Metadata: 0
+Customs:
+  Item: writable_book
```
Comment (PyvesB) [inline src/main/resources/gui.yml]: This is not a valid Material name, I'm afraid the plugin is complaining whilst starting up.

File: src/main/resources/plugin.yml [hunk 4]
```diff
@@ -193,6 +196,6 @@ permissions:
   achievement.count.smelting:
     description: Allows smelting statistics in database to increase.
     default: true
-# Additional permissions are available for the Breaks, Places, Crafts, Kills, Breeding and PlayerCommands categories.
+# Additional permissions are available for the Breaks, Places, Crafts, Kills, Breeding, PlayerCommands and Customs categories.
```
Comment (PyvesB) [inline src/main/resources/plugin.yml]: Good catch! 😃

**Comment by Phoetrix** [comment]
Oh good! I was afraid that you don't like my idea. ^^

Thank you! I'm learning a lot of interesting things with your corrections. :D
I'm sorry for the "Item: writable_book". I would like to put "book_and_quill" but I selected the id from Minecraft and not the Material name. ^^' I saw that "book_and_quill" is already taken so I just set "feather".

Now it's better! :)

File: src/main/resources/lang.yml [hunk 5]
```diff
@@ -38,6 +38,11 @@ achievement-already-received: "The player PLAYER has already received this achie
 achievement-not-found: "The specified achievement was not found in Commands category."
 achievement-given: "Achievement given!"
 
+# Related to /aach add.
+error-value: "The value VALUE must to be an integer!"
+achievement-increase: "Achievement ACH increase by AMOUNT for PLAYER!"
```
Comment (PyvesB) [inline src/main/resources/lang.yml]: Should we talk about "statistics" here as well? Also I would have written "increase**d**".

File: src/main/resources/lang.yml [hunk 6]
```diff
@@ -171,6 +178,7 @@ list-petmastergive: "Pets Given to Another Player"
 list-petmasterreceive: "Pets Received from Another Player"
 list-smelting: "Items Smelt"
 list-commands: "Other Achievements"
+list-customs: "Customs Achievements"
```
Comment (PyvesB) [inline src/main/resources/lang.yml]: For players, the difference between "Other Achievements" and "Customs Achievements" may be confusing. What do you think about "Other Achievements" and "Custom Categories" to make things a little clearer?

**Comment by Phoetrix** [comment]
Yes I agree! :)
But I don't know what I'm supposed to do with CommandTabCompleter. I prefer to let you do, thanks! ^^

**Comment by PyvesB** [comment]
Okay, no problem. Merged, thanks again for your contribution! 😉

**Comment by PyvesB** [comment]
As a side note before I forget, next time I recommend that you leave your `master` branch as an exact copy of the main repository, and that you develop your pull request on a different branch (you can open a pull request from any branch). As you kept building on top of your own `master`, the commits from your previous pull request were resubmitted in this one as well (but were automatically merged). ^^

**Comment by PyvesB** [comment]
@Phoetrix : I've updated the [Wiki](https://github.com/PyvesB/AdvancedAchievements/wiki/Commands-and-permissions) with the new _/aach add_ command, feel free to edit if I've missed something!

**Pull request merged**